core/includes/bootstrap.inc

```diff
@@ -726,7 +726,7 @@ function backdrop_environment_initialize() {
 
   // When clean URLs are enabled, emulate ?q=foo/bar using REQUEST_URI. It is
   // not possible to append the query string using mod_rewrite without the B
-  // flag (this was added in Apache 2.2.8), because mod_rewrite unescapes the
+  // flag (this was added in Apache 2.2.8), because mod_rewrite un-escapes the
```

Comment:
@BWPanda this seems to be legit: https://en.wiktionary.org/wiki/unescape

So perhaps add it to the Backdrop-specific dictionary and allow it?

Reply:
I would trust more the Merriam-Webster dictionary, which say [*unescape* does not exist](https://www.merriam-webster.com/dictionary/unescape). 😅

Reply:
Since this is a programming term, and since at least one other place on the internet uses it and considers it a word, let's use it and I'll add it to Backdrop's dictionary.